coderd/database/dbmem/dbmem.go

```diff
@@ -723,18 +723,68 @@ func (q *FakeQuerier) getOrganizationMemberNoLock(orgID uuid.UUID) []database.Or
 	return members
 }
 
+var ErrUserDeleted = xerrors.New("user deleted")
```

Comment:
This error is never thrown outside dbmem right? We don't need to export it outside the package.
```suggestion
var errUserDeleted = xerrors.New("user deleted")
```

Reply:
Good catch.